src/main/java/com/iexec/commons/containers/client/DockerClientInstance.java

```diff
@@ -688,7 +687,12 @@ public int waitContainerUntilExitOrTimeout(
             if (seconds % 60 == 0) { // don't display logs too often
                 log.info("Container is running [name:{}]", containerName);
             }
-            WaitUtils.sleep(1);
+            try {
+                Thread.sleep(1000);
```

Comment:
What about the following? It's less confusing, as the unit is explicitly mentioned.
```suggestion
                TimeUnit.SECONDS.sleep(1);
```